src/rag_processor/auth/cloudflare.py

```
@@ -0,0 +1,332 @@
+Supports bypass mode for local development without Cloudflare.
+"""
+
+from __future__ import annotations
+
+import time
+from datetime import UTC, datetime
+from typing import TYPE_CHECKING, Any, ClassVar
+
+import httpx
+import jwt
+from cryptography.hazmat.primitives.asymmetric.rsa import RSAPublicKey  # noqa: TC002
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `TC002`)

Remove unused `noqa` directive

(RUF100)